quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/themes/FlatClasspathThemeResourceProviderFactory.java

```diff
@@ -19,9 +19,15 @@
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
 import java.net.URL;
+import java.nio.charset.Charset;
 import java.util.Enumeration;
+import java.util.Locale;
+import java.util.Properties;
 import org.keycloak.theme.ClasspathThemeResourceProviderFactory;
+import org.keycloak.theme.PropertiesUtil;
```

Comment:
could you please remove the unused imports here? The code that used these seems to be in [ClasspathThemeResourceProviderFactory.java](https://github.com/keycloak/keycloak/pull/11711/files#diff-7e5492140ee2d42c1e7456d9bed8ce8d61598d76410118b5eeb50510921c7874) now.